daml-assistant/daml-project-config/DAML/Project/Types.hs

```diff
@@ -85,6 +85,11 @@ newtype DamlPath = DamlPath
     { unwrapDamlPath :: FilePath
     } deriving (Eq, Show)
 
+-- | Absolute file path to the assistant.
+newtype DamlAssistantPath = DamlAssistantPath
```

Comment:
Could we be clear whether it is the dir that daml assistant lives in, or the full path with .exe etc on the end? Ditto all the other ones in this module.

Reply:
I’ve tried to make it more clear and added an example for this variable. I’ll leave the other ones for separate PRs.

Reply:
👍